### DISCUSSION TITLE
스크린샷 기능이 모바일에선 정상 작동하질 않습니다

### DISCUSSION THREAD

**Discussion by munhyok** [naverz/zepeto-studio-kor#928] [5. Q&A]
스크린샷 기능을 만드려고 예제를 따라 만들었는데
유니티에디터에선 상당히 잘되었는데
![image](https://user-images.githubusercontent.com/11683617/176024869-38a6a495-c3e3-4913-b116-bdbcb3289001.png)
Post To Feed를 눌렀을 때 로그입니다..

모바일에선 시점 변경과 촬영버튼 둘중 하나를 누르면 아예 작동이 되질 않고
둘중에 하나라도 누르고 X 아이콘을 누르면 캐릭터 3인칭으로 다시 돌아오지 않습니다...
튜토리얼을 다시 보고 예제 파일을 봐도 놓친게 없는거같은데... 왜 이러는지 잘 모르겠네요...

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 멀티플레이 샘플을 받아서 실행할 때는 모바일에서도 잘 되는데
새로운 프로젝트에서 구현하니까 관련 오류가 발생하신 상황인지 확인 부탁드립니다.

**Reply by Zepeto-TechEvangelist** [reply]
https://github.com/naverz/zepeto-studio-kor/discussions/581#discussioncomment-2506605
유사한 이슈 히스토리 참고 부탁 드립니다.
